analysis/create_tables.R

```diff
@@ -53,31 +53,45 @@ readr::write_csv(
   here::here("output", "population", "pf_demographics.csv")
 )
 
-# gt_table <- df_demographics_counts[1:2] %>%
+# Demographics table with percentages
+df_demographics_table <- head(df_demographics_counts, 39) %>%
```

Comment:
This feels like an odd way to select the first 39 rows, is there a more robust way of selecting the correct rows here?